ogre_mesh_viewer.py

```diff
@@ -460,6 +481,18 @@ def _save_screenshot(self):
         self.getRenderWindow().writeContentsToTimestampedFile(outpath, ".png")
         self.cam.getViewport().setOverlaysEnabled(True)
 
+    def _open_file_dialog(self):
```

Comment:
I know you have already sunk many hours into this, however it is unfortunately still incomplete:
- resource locations should be updated. if `self.filedir` is different after "Open", then loading fails
- the `draw_loading` is probably broken. (shown if you import a 40MB .obj file)
- I could trigger crashes when repeatedly opening a sekeletal animated mesh

what do you think about merging the initial file dialog first, and leaving the "Open" menu for later?

Reply:
> I know you have already sunk many hours into this, however it is unfortunately still incomplete:
> 
> * resource locations should be updated. if `self.filedir` is different after "Open", then loading fails
> * the `draw_loading` is probably broken. (shown if you import a 40MB .obj file)
> * I could trigger crashes when repeatedly opening a sekeletal animated mesh
> 
> what do you think about merging the initial file dialog first, and leaving the "Open" menu for later?

Let's see if I can find a solution to these issues

Reply:
I think that all these issues have been addressed.

Let me know